### ISSUE TITLE
An error occurred Error 500 (Server Error): Internal server error  Endpoint: /changes/

### ISSUE THREAD

**Issue by learningendless** [GerritCodeReview/docker-gerrit#17]
I made a new gerrit site according the docker hub readme: https://hub.docker.com/r/gerritcodereview/gerrit . There is no error and warning during initializing gerrit and starting all containers.
But I got the following error when I access gerrit site with chrome and firefox browers.
Is there anyting should be updated in the docker hub readme?(especially after using h2 instead of postgresql, and replacing ReviewDB with NoteDB)

An error occurred
Error 500 (Server Error): Internal server error
Endpoint: /changes/

**Comment by dpursehouse** [comment]
Please check the logs on the server to find any exception logs, and then submit a bug report at https://bugs.chromium.org/p/gerrit/issues/list

**Comment by learningendless** [comment]
@dpursehouse I checked the gerrit log, and I have fixed this issue. There are some failed error in the migration process(from reviewdb to notedb).
I was creating a new gerrit site, so I change the file gerrit.config to use noteDb rather than the readme on docker hub.
following:
external/gerrit/etc/gerrit.config
[database]
  type = h2
  database = db/ReviewDB     >>>>   database = db/noteDb
I cannot make sure if the word "noteDb" is right. but it worked.
Advise you update the github readme and docker hub readme.
